docs/en/setup/backend/backend-graalvm.md

```diff
@@ -46,4 +46,12 @@ we can successfully start SkyWalking-oap.
 With native-image, some features are not yet supported.
 
 1. [LAL](https://skywalking.apache.org/docs/main/next/en/concepts-and-designs/lal/), [MAL](https://skywalking.apache.org/docs/main/next/en/concepts-and-designs/mal/), and some other features related to them are not supported at the moment.
-2. The [OAL](https://skywalking.apache.org/docs/main/next/en/concepts-and-designs/oal/) files are used in the compiling stage, which means that users would not see these files inside the native image package, and can't change it. Consider recompiling and packaging from the source codes including your OAL file changes.
\ No newline at end of file
+2. The [OAL](https://skywalking.apache.org/docs/main/next/en/concepts-and-designs/oal/) files are used in the compiling stage, which means that users would not see these files inside the native image package, and can't change it. Consider recompiling and packaging from the source codes including your OAL file changes.
+
+## Current Limitations
+Native-image supports reflection and other dynamic features through some JSON-formatted configuration files. SkyWalking currently provides a set of configuration files for basic support. You can find them [here](https://github.com/apache/skywalking/tree/master/graal/graal-server-starter/src/main/resources/META-INF/native-image).
+
+For now, these configuration files do not support all runtime environments (but will be fully supported in the future). Therefore, in other environments, users may need to generate the configuration files required by native-image on their own.
```

Comment:
As we discussed, we should have a new maven plugin to scan give classes by names, annotations, package path, etc.

We should get rid of the native build agent, as generally, the SkyWalking OAP is modularized, we can't guarantee the codes of reflect have been invoked.